graviton/blackbox.py

```diff
@@ -49,7 +46,9 @@ class ExecutionMode(Enum):
 
 
 class DryRunProperty(Enum):
-    cost = auto()
+    cost = auto()  # deprecated
```

Comment:
@tzaffi Optional:  Since `DryRunProperty.cost` is retained as a derived property, I think the deprecated comment can be removed.  As I understand it, the graviton property is _not_ deprecated, only algod's response field named `cost`.

```suggestion
    cost = auto()
```